Call show()

Fatal error: Uncaught Error: Call to private method pass::show() from context 'fail' in %s:%d
Stack trace:
#0 %s(%d): fail::not_ok()
#1 {main}
  thrown in %s on line %d